src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
+        String username = "";
         try {
             user = authenticationService.login("bob", "1234");
+            username = user.getLogin();
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't login a user: {}", username, e);
```

Comment:
The log message in the `catch` block could be more informative. According to the checklist, log messages should include relevant parameters. Consider including more details about the exception or the context in which it occurred.